src/fillalgebra.jl

```diff
@@ -520,3 +520,12 @@ function kron(f::AbstractFillVecOrMat, g::AbstractFillVecOrMat)
     sz = _kronsize(f, g)
     return _kron(f, g, sz)
 end
+
+# bandedness
+LinearAlgebra.isdiag(A::AbstractFillMatrix) = iszero(getindex_value(A)) || isempty(A)
```

Comment:
need to also check if its 1x1

Reply:
Turns out that this specialization isn't necessary, as the `istriu` and `istril` ones automatically handle this